integration_tests/print_02.py

```diff
@@ -11,10 +11,13 @@ def f():
     print(b)
     print(c)
     print(d)
+    print(a, a, b, c, d)
```

Comment:
What about `print(a, a, 1, b, c, 1.1, d)`?

Reply:
Yep, does work:
```py
from ltypes import i32, f64

def f():
    a: list[str] = ["ab", "abc", "abcd"]
    b: list[i32] = [1, 2, 3, 4]
    c: list[f64] = [1.23 , 324.3 , 56.431, 90.5, 34.1]
    d: list[i32] = []

    print(a, a, 1, b, c, 1.1, d)

f()
```
Output:
```console
% lpython a.py 
['ab', 'abc', 'abcd']
['ab', 'abc', 'abcd']
1
[1, 2, 3, 4]
[1.22999999999999998e+00, 3.24300000000000011e+02, 5.64309999999999974e+01, 9.05000000000000000e+01, 3.41000000000000014e+01]
1.10000000000000009e+00
[]
```

Reply:
Great. Add it as a test.